.bumpversion.cfg

```diff
@@ -0,0 +1,14 @@
+[bumpversion]
+commit = True
+tag = True
+message = "Release {new_version}"
+current_version = 0.1.0
```

Comment:
0.0.0 allows you to make any release you want later, 0.0.1 suggests it's already ready, and if you'd have errors.... :dancers: